test/integration/users.test.js

```diff
@@ -38,6 +39,10 @@ describe("Users", function () {
     jwt = authService.generateAuthToken({ userId });
     superUserId = await addUser(superUser);
     superUserAuthToken = authService.generateAuthToken({ userId: superUserId });
+
+    const docRefUser0 = photoVerificationModel.doc();
```

Comment:
Wondering if this number needs to be part of the variable name.

Reply:
Changed it